### PR TITLE
Some fixes on pagination, nullable fields, and other misc

### PR DESCRIPTION
**Pull request by gilbertfu**
- New card payment type
- new pan account number type
- new ref number types
- add back documents to payment order req (with description for using diff content type)
- nullable receiving_account on payment order
- fix pagination for transactions

### CODE REVIEW

**Review by lukecivantos — COMMENTED** [review]
Left some questions below but otherwise lgtm – also mostly curious what is the new card payment type?

File: openapi/mt_openapi_spec_v1.yaml [hunk 1]
```diff
@@ -4257,7 +4294,9 @@ components:
             additionalProperties: false
             minProperties: 16
           - "$ref": "#/components/schemas/internal_account"
+          - type: 'null'
           description: The receiving account. Can be an `external_account` or `internal_account`.
```
[the remark this answers is not in the document]
Comment (gilbertfu) [reply openapi/mt_openapi_spec_v1.yaml:4298]: yea removed the unnecessary type null, nullable true is sufficient.

File: openapi/mt_openapi_spec_v1.yaml [hunk 2]
```diff
@@ -2800,7 +2830,7 @@ components:
         documentable_type:
           type: string
           enum:
-          - compliance/case
```
[the remark this answers is not in the document]
Comment (gilbertfu) [reply openapi/mt_openapi_spec_v1.yaml:2803]: this looks right

[the remark this answers is not in the document]
Comment (mchaudhry05) [reply openapi/mt_openapi_spec_v1.yaml:2803]: yup this is good! we scoped it to just be `case` and not `compliance/case`

[the remark this answers is not in the document]
Comment (lukecivantos) [reply openapi/mt_openapi_spec_v1.yaml:2803]: Ahh awesome didn't see that update – probably makes a lot of the routing easier 😅

**Review by lukecivantos — APPROVED** [review]
lgtm!

**Pull request merged**